### PR TITLE
starting some docs for our next meeting discussions

### PR DESCRIPTION
**Pull request by bnlawrence**
These docs may or may not yet render as proper diagrams

### CODE REVIEW

File: docs4understanding/active.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+If however, it is possble to break down the logic of `_from_storage` so that individual "part_chunks" which are contiguous are processed on the storage side of the LAN (in the ost or nearby) then meaningful performance is possible. The Python client in the end would simply see a list of partial products of the computational method which have come direct from the osts. It will not care whether those parts came from a different breakdown of storage than it anticpated in the chunking (though of course the storage will need to do the necessary mapping to generate the partial sums).
+
+In the longer term, where we expect that we will have to pass a decompression method down through the `_decode_chunk` interface, it _will_ be necessary for the computational storage to respect the `_decode_chunk` interface server-side. This is of course what is required with S3, where we effectively need an S3 proxy to do the work. It might be that this is also required in some implementations of POSIX storage systems if they wish to implement computational storage OR they will need to somehow respect block contiguous placement in some way.
+
```
Comment (valeriupredoi) [inline docs4understanding/active.md:68]: @bnlawrence I think this is probably better suited in the README?

**Review by valeriupredoi — CHANGES_REQUESTED** [review]

**Comment by valeriupredoi** [comment]
cheers for this @bnlawrence - I think any operational example should probably go in the README for now, before we actually get sphinx-ed dox, feel free to say no to that - I also requested changes to get you used with a GH review that requests changes (in case you're not used to, am not sure of your GH user level, apols if I grossly underestimated it :grin: )

File: docs4understanding/active.md (hunk 1 above)
Comment (bnlawrence) [reply docs4understanding/active.md:68]: Sure, but for the moment this is intended for discussion this Friday ...

**Comment by bnlawrence** [comment]
(Not possible to underestimate my GH foo)

File: docs4understanding/active.md (hunk 1 above)
Comment (valeriupredoi) [reply docs4understanding/active.md:68]: ah okay then it's fine to have it in the pseudo-docs repo, cheers @bnlawrence

**Review by valeriupredoi — APPROVED** [review]

**Pull request merged**